corehq/apps/users/models.py

```diff
@@ -2050,6 +2055,12 @@ def __repr__(self):
             self=self
         ))
 
+    @skippable_quickcache(['self._id'], lambda _: settings.UNIT_TESTING)
+    def get_usercase_id(self):
+        from corehq.apps.hqcase.utils import get_case_by_domain_hq_user_id
+        usercase = get_case_by_domain_hq_user_id(self.domain, self._id, USERCASE_TYPE)
```

Comment:
Sorry for chiming in late but it seems silly to load the whole case if we only want the ID